src/v2/Apps/Consign/Routes/MarketingLanding/Components/ConsignMeta.tsx

```diff
@@ -1,11 +1,11 @@
-import * as React from "react";
+import * as React from "react"
 import { MetaTags } from "v2/Components/MetaTags"
 
 export const ConsignMeta: React.FC = () => {
   return (
     <MetaTags
       pathname="/consign"
-      title="Sell Artwork with Artsy | Art Consignment | Artsy"
```

Comment:
Can we use `Sell Artwork with Artsy | Artsy` here?

Reply:
product asked for 'Sell Artwork with Artsy' only. here: https://docs.google.com/document/d/1CyIph7Iqts-6nV9YNhV98ejQniLTQ45x9SQokFeMP3g/edit